### DISCUSSION TITLE
天选时刻/实物抽奖/预约抽奖屏蔽词收集

### DISCUSSION THREAD

**Discussion by andywang425** [andywang425/BLTH#80] [Information Gathering]
欢迎各位在这里分享你的天选时刻/实物抽奖/预约抽奖忽略关键字，我会挑选一部分添加到默认设置或上传到云端，其他用户可通过点击脚本中的【添加云端关键字】按钮来添加这些屏蔽词。  
云端上已有的屏蔽词可点击该链接查看：https://github.com/andywang425/BLTH/blob/master/assets/json/notice.json  
各位只需复制你的【忽略关键字】然后全部发上来即可，无需做去重处理。

**Comment by nankewei** [comment]
测试,钓鱼,炸鱼,大航海,上船,舰长,返现,抵用,代金,黑屋,上车,上反船,自拍,照片,照,自画像,画头像,写真,第五人格,星际战甲,保温服卡,qq群,相册,皂片,M4A1,星光赏,怒火燎原,宠物,高考,闪光狼人,两元红包,cos,取关,加速器月卡CDK,上舰,礼花以内免费点播,王者带你上一颗星,航海船票,船票,船,1-5元红包,幸运车车,qq号,壁纸,马克杯1/5,微信红包封面,随机皮肤礼包,点播,加速器,天使之赐(游戏道具),素颜,开舰,封面,美照,随机照,女装照,日常照,好友,给主播,照骗,连麦,情书,一局,舰涨优惠卷,一起玩,开黑,3天角色卡,点歌,吃席,车车一把,陪打游戏,主播人设原图png,尖叫,总督,胶囊

**Comment by Marina17700** [comment]
测试,钓鱼,炸鱼,大航海,上船,舰长,返现,抵用,代金,黑屋,上车,总督,提督,照片,素颜,cos,封面,写真,自拍,签名,自画像,key,挂件,照骗,明信片,手机壳,皂片,皮肤,好友,女装照,日常照,头像,jk照,主播,上舰,点播,电子版日历,《,私房照,生活照,壁纸,猫片,排位,单拉,一起打游戏,开黑,短片,陪打游戏,一起玩,3天角色卡,点歌,吃席,车车,点歌,体验卡,内测码,cdk,第五人格,一局

**Comment by LuluDeer** [comment]
测试,钓鱼,炸鱼,黑屋,脚本,空气,开通,大航海,上船,舰长,返现,抵用,代金,礼金,上车,上反船,jk照,照片,图包,短片,素颜,自拍,写真,皂片,开舰,上舰,自画像,封面,取关,美照,随机照,女装照,日常照,好友,给主播,照骗,连麦,情书,名字,一局,舰涨,优惠卷,一起玩,开黑,尖叫,点歌,总督,胶囊,壁纸,吃席,点播,船长,/^\d+$/,/写.*名字/,/给.*美甲/,陪打,/带.*上.*星/

**Comment by nankewei** [comment]
定制团子,饮食分析/情感解惑x1,饮食分析,情感解惑,小红宝,大蒜

**Comment by xingchenyuqing** [comment]
舰长,照,总督,提督,一毛,0.01,禁言,0.0,代金,片,通行证,第五人格,抵用禁言,脸,越南,吃席,体验,小红宝,陪,和平精英衣服,一瓶,主播,人设,开明,完美偶像,可乐红包,项目,橙子,橘子,账号,规划,紫薯,亲亲,海报,管理,房间,紫皮,角色,线索,贺卡,大鸟,签名,带,礼花,一起,沉默,笔芯,网课,相册,一本,车车,JZ,玩游戏,手写卡片,徽章,闪蛋,位,卢布,zp,船,木马,撒娇,取关,带你,1-5,小布丁,solo,符,渔夫,锄地,米哦,情书,歌,壁纸,胡萝卜,给,看号,画,红花油,warframe,涂装,麦霸,内战,头像,加速器,螺旋,深渊,碎片,挂件,辣条,女装,点播,图,表情,镜子,大蒜,邮费,自付,背景,带人,学习资料,钥匙扣,现场验证,抵用,虚拟,提示音,钥匙,cos,若奇,图包,测试,钓鱼,炸鱼,黑屋,上车,柑果球,照,喵娘抽奖,一起玩,明信片,皂片,总裁,阵容,唇印,早起,晚安,铃声,语音,点歌,吻,陪你玩,极巨大冒险,宝可梦,小游戏,穿越火线,保温服卡,星际战甲,第五人格,许愿码,大航海,双排,五排,排位,优惠券,代金,抵扣,返现,好友,一局,房管,房官,开黑,总督,提督,续舰,双马尾,图片,自拍,自画,写真,封面,素颜,拉黑,黑名单,抽奖号,test,加密,舰,贴纸,船票,M4A1,一块玩,相片,自画像,反船,无脸照,cos照,车位,网剧,cos自拍,代打,写真照,一次,专属头衔,照片,笔记本,游戏,手游,皮肤,宠物,空气,蒸汽,月饼,加速,美照,舰涨,舰长,上舰,cdk,上船,月历,源石,旦酱,流连,红酒,小黑屋,手工,尖叫,宝石,钻石,充值卡,券,脚本,文件夹,电池,赛车,虚拟奖品,保底,代币,刀币,课,局,越南盾,网点,机器,话梅,志愿,令牌,铜牌,皇冠,唢呐

**Comment by BB-GOAT** [comment]
【腾讯文档】B站天选抽奖屏蔽词列表（可以参考，持续更新）
https://docs.qq.com/doc/DWFR0VVJKdWRKQktR
太多了，直接搬文档链接算了←v←

**Comment by zkzssf** [comment]
和主播一起打游戏,主播私拉链接一把,点歌券,车车一局,优惠券

**Comment by ZiuChen** [comment]
签名照 :confused:

**Comment by PODF-x** [comment]
舰,船,卷,照片,自拍

**Comment by xxblog** [comment]
测试,炸鱼,钓鱼,黑屋,脚本,空气,test,cdk,gdk,cos,dlc,key,jk,/([zh]ao|xian(g)?)pian(g)?/,舰,船,航,督,图,照,脸,车,車,/(相|短|皂|猫)片/,摄影作品,手写卡片,表白信,手写信,真心话,明信片,表情包,画像,海报,相框,封面,自拍,写真,壁纸,头像,素颜,点歌,点播,连麦,排位,/.排/,开黑,情感解惑,饮食分析,游戏位,/[ch]e位/,和主播,一起,一局,陪打,代练,代打,代抽,看号,保底,好友,/(激活|内测|许愿|体验|充值|礼包|立减|红包|点)(码|卡|卷|劵|券)/,不折现,返现,抵用,抵扣,代金,礼金,兑换,优惠,资格,/(钻|宝|源|原)石/,/(金|代|刀)币/,网点,摩啦,摩拉,晶核,第五人格,星际战甲,原神,生死狙击,时空猎人,王者荣耀,穿越火线,打深渊,阴阳师,加速器,崩坏,王者

**Comment by Mishasama** [comment]
![image](https://user-images.githubusercontent.com/54498172/180591251-33150f09-d82c-4b9f-8d2b-d60348914647.png)

棒棒糖
